shell/edit/provisioning.cattle.io.cluster/MachinePool.vue

```diff
@@ -64,11 +64,17 @@ export default {
 
   data() {
     const parseDuration = (duration) => {
-      // The back end stores the timeout in Duration format, for example, "10m".
-      // Here we convert that string to an integer.
-      const numberString = duration.split('m')[0];
+      // The back end stores the timeout in Duration format, for example, "10m30s".
+      // Here we convert that string to an integer and return the duration as seconds.
+      const splitStr = duration.split(/[m,s]/).filter(s => s);
```

Comment:
This seems to return the same array as-is - can it be removed? 

```
.filter(s => s)
```

Reply:
I had this to remove the empty string at the end of the array, but it isn't really necessary and can be removed.